apis/intermediate_api/intermediate_api/api/project_api.py

```diff
@@ -6,7 +6,35 @@
 import intermediate_api.decorators as decorators
 import intermediate_api.auth0 as auth0
 import intermediate_api.constants as const
-import intermediate_api.api.intermediate_api as intermediate_api
+
+
+def has_project_permission(is_authenticated):
```

Comment:
Thoughts on making this a decorator?

Reply:
Actually, yes, I think this can be done via decorator as by the time the API gets hit, `@app.route()` is the very first one which includes `request`, I originally thought of this as a decorator but because of that `request` variable, I was trying to do it within each endpoint.